### ISSUE TITLE
System.ArgumentException: An item with the same key has already been added. Key: arg => arg

### ISSUE THREAD

**Issue by idzmitry** [epam/epam-graphql#95]
### Expected Behavior

First 2 concurrent queries to a same _MutableLoader_ do not fail.

### Current Behavior

Right after start of an app **first 2** concurrent queries to a same _MutableLoader_ fail due to error _ArgumentException: An item with the same key has already been added. Key: arg => arg_
```
Stack trace:
       at System.Collections.Generic.Dictionary`2.TryInsert(TKey key, TValue value, InsertionBehavior behavior)
       at System.Collections.Generic.Dictionary`2.Add(TKey key, TValue value)
       at Epam.GraphQL.Configuration.Implementations.ProxyAccessor`2.<Configure>g__AddNewField|26_0(String fieldPrefix, LambdaExpression depExpr, <>c__DisplayClass26_0& ) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\ProxyAccessor.cs:line 158
       at Epam.GraphQL.Configuration.Implementations.ProxyAccessor`2.Configure() in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\ProxyAccessor.cs:line 109
       at Epam.GraphQL.Configuration.Implementations.ProxyAccessor`2.get_ProxyGenericType() in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\ProxyAccessor.cs:line 52
       at Epam.GraphQL.Configuration.Implementations.ProxyAccessor`2.<GetConcreteProxyType>b__43_0(ICollection`1 fieldNames) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\ProxyAccessor.cs:line 379
       at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
       at Epam.GraphQL.Configuration.Implementations.ProxyAccessor`2.GetConcreteProxyType(IEnumerable`1 fieldNames) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\ProxyAccessor.cs:line 377
       at Epam.GraphQL.Configuration.Implementations.ProxyAccessor`2.<CreateSelectorExpression>b__29_0(ICollection`1 fieldNames) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\ProxyAccessor.cs:line 209
       at System.Collections.Concurrent.ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
       at Epam.GraphQL.Configuration.Implementations.ProxyAccessor`2.CreateSelectorExpression(IEnumerable`1 fieldNames) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\ProxyAccessor.cs:line 204
       at Epam.GraphQL.Configuration.Implementations.FieldResolvers.QueryableFuncResolver`3.Transform(IResolveFieldContext context) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\FieldResolvers\QueryableFuncResolver.cs:line 248
       at Epam.GraphQL.Configuration.Implementations.FieldResolvers.QueryableFuncResolver`3.<>c__DisplayClass17_0.<AsConnection>b__1(IResolveFieldContext ctx) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\FieldResolvers\QueryableFuncResolver.cs:line 219
       at Epam.GraphQL.Configuration.Implementations.FieldResolvers.FuncResolver`3.Resolve(IResolveFieldContext context) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Configuration\Implementations\FieldResolvers\FuncResolver.cs:line 37
       at GraphQL.Execution.ExecutionStrategy.ExecuteNodeAsync(ExecutionContext context, ExecutionNode node) in /_/src/GraphQL/Execution/ExecutionStrategy.cs:line 209
       at GraphQL.Execution.SerialExecutionStrategy.ExecuteNodeTreeAsync(ExecutionContext context, ObjectExecutionNode rootNode) in /_/src/GraphQL/Execution/SerialExecutionStrategy.cs:line 38
       at GraphQL.Execution.ExecutionStrategy.ExecuteAsync(ExecutionContext context) in /_/src/GraphQL/Execution/ExecutionStrategy.cs:line 33
       at GraphQL.DocumentExecuter.ExecuteAsync(ExecutionOptions options) in /_/src/GraphQL/Execution/DocumentExecuter.cs:line 240
       at Epam.GraphQL.SchemaExecuter`1.ExecuteAsync(SchemaExecutionOptions`1 schemaExecutionOptions) in C:\Users\idzmitry\Desktop\epm\epam-graphql\src\Epam.GraphQL\Schema\SchemaExecuter.cs:line 71
       at Learn.Web.Controllers.EpamGraphQlController.Query(GraphQLRequest request) in C:\Users\idzmitry\Desktop\epm\learn\Learn.Web\Controllers\EpamGraphQlController.cs:line 49
       at lambda_method(Closure , Object )
       at Microsoft.Extensions.Internal.ObjectMethodExecutorAwaitable.Awaiter.GetResult()
       at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
       at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Logged|12_1(ControllerActionInvoker invoker)
       at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
       at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
       at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
       at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.InvokeInnerFilterAsync()
```
Probably there is another related issue in _ProxyAccessor_ with stack trace pointing in the same place. Please see the screenshot:
 
![error](https://user-images.githubusercontent.com/22106087/187704666-08598f83-463e-4c1d-8e65-e2942abbea3f.png)

### Steps to reproduce

1. Define 1 _MutableLoader_
2. Start app
3. Run 2 concurrent requests to the loader

**Comment by idzmitry** [comment]
As I understand `Configure` is executed only once during application lifecycle per each loader, so usage of `lock` won't hit performance.

**Comment by kuzn-ilya** [comment]
Fixed in 2.4.0
